src/event.cpp

```diff
@@ -116,7 +116,9 @@ void event::actualize()
         if (g->u.has_amount("petrified_eye", 1)) {
```

Comment:
I'd move the check earlier:

`if (g->u.has_amount( "petrified_eye", 1 ) ) && !g->u.is_deaf() ) {`

Otherwise how do you know the eye made a sound if your deaf?